### ISSUE TITLE
Keyboard shortcut for select text in the entry field of the custom search engine

### ISSUE THREAD

**Issue by Kristinita** [ccampbell/mousetrap#360]
I have [my own search engine HTML page](http://kristinita.ru/technical_texts/TestSearch). Code of this page:

    <script>
      (function() {
        var cx = '008959915961530481423:nigd_brnqzu';
        var gcse = document.createElement('script');
        gcse.type = 'text/javascript';
        gcse.async = true;
        gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
        var s = document.getElementsByTagName('script')[0];
        s.parentNode.insertBefore(gcse, s);
      })();
    </script>
    <gcse:search></gcse:search>

It is necessary for me that users from any browser could put by means of a shortcut — for example, <kbd>Ctrl+Alt+K</kbd>, — the cursor in the entry field of my search engine.

![cursor in input field](http://i.imgur.com/a7HElSl.png)

Is it possible to do select the text in the input field of my search engine using a mousetrap?

Thanks.

**Comment by ccampbell** [comment]
Take a look at: https://craig.is/killing/mice#api.bind.text-fields

Is that what you are looking for?

**Comment by Kristinita** [comment]
@ccampbell , no. In my code it contains only a script that I put in my post above, and does not contain `input` or `textarea` tags. I do not know how can I create a shortcut to select the text in the input field. Is it possible, using a mousetrap?

Thanks.
